### PR TITLE
Qualifying DC/OS 1.11.11 against RHEL 7.6

### PR DESCRIPTION
**Pull request by ahairape**
JIRA: https://jira.mesosphere.com/browse/DCOS-53920

### CODE REVIEW

**Comment by cprovencher** [comment]
why remove all those other qualified combinations?

**Comment by ahairape** [comment]
@cprovencher could you please have a look at the error message for this PR? I have tried to apply the changes from your PR https://github.com/dcos/dcos-images/pull/232 to RHEL 7.6 base OS.
When I am setting `enable_os_setup_script = false` here (https://github.com/dcos/dcos-images/pull/236/files#diff-2bb732749e4759687c328f8e0c6de8f7R18), I am getting `docker should be installed and running` error, e.g.: https://jenkins.mesosphere.com/service/jenkins/blue/organizations/jenkins/dcos-images/detail/PR-236/17/pipeline#step-93-log-1238

File: build_test_publish_images.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 TEST_CONFIG_YAML = 'publish_and_test_config.yaml'
 
-TERRAFORM_DCOS_VERSION_PIN = "22583e002848f77c2ac403b125fe7e22f83d10fd"
+TERRAFORM_DCOS_VERSION_PIN = "34a873ab27462a9a2c016034d2289357182d727a"
```
Comment (gauripowale) [inline build_test_publish_images.py]: Is there a reason you changed the version of terraform dcos that is pinned?

Comment (ahairape) [reply build_test_publish_images.py]: No specific reason, I have just taken the latest available commit.

**Pull request merged**